### DISCUSSION TITLE
Process problem of two-phase flow simulation (Predicting relative permeability) tutorial

### DISCUSSION THREAD

**Discussion by likeduck** [PMEAL/OpenPNM#2722] [Q&A]
Hi Prof, 
I meet a question in the `Predicting relative permeability` in openpnm website.
I followed all the code and only replaced the PNM, the result is very satisfactory, but only one chance. I changed the surface tension, contact angle of air, but the simulation results ( curve ) will not change. I don 't know where the problem is, can you tell me the possible reason？（The probable reason is that I don 't know where to use `regenarate _ model`.）
Thank you very much！
The only curve is：
![non_orangic](https://user-images.githubusercontent.com/117707506/229402106-a61b13d7-be97-4ad5-847e-8c828004d202.png)

**Comment by jgostick** [comment]
If you are only changing things like surface tension and contact angle, then nothing will change in the relative permability curve because the invasion configuration does not change.  To see changes in Rel Perm you much change something that will result in a different invaison pattern.  If you apply random contact angles within a certain range then the curves should change, for instance.

**Reply by likeduck** [reply]
Yes! I have the idea that set anisotropic pores and throats but it also under implementation. Thank you so much!
